bindings/imap-codec-python/examples/common/__init__.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from enum import Enum
+from typing import Optional
+
+COLOR_SERVER = "\x1b[34m"
+COLOR_CLIENT = "\x1b[31m"
+RESET = "\x1b[0m"
+
+
+class Role(Enum):
+    Client = 1
+    Server = 2
+
+
+def read_more(buffer: bytearray, role: Role):
+    if not buffer:
+        prompt = "C: " if role == Role.Client else "S: "
+    else:
+        prompt = ".. "
+
+    line = read_line(prompt, role)
+
+    # If `read_line` returns `None`, standard input has been closed.
+    if line is None or line.strip() == "exit":
```

Comment:
Nope. Should be okay.

Reply:
Well, there is that:
```
Enter IMAP command (or "exit").

C: a SELECT {4}

S: +
.. quit 

{'tag': 'a', 'body': {'Select': {'mailbox': {'Other': {'String': {'Literal': {'data': [113, 117, 105, 116], 'mode': 'Sync'}}}}}}}
C: a SELECT {4}

S: +
.. exit

Exiting.
```

However, I would argue this is nothing we should try to prevent in the examples as these are more or less academic anyways.